apis_core/apis_metainfo/models.py

```diff
@@ -88,6 +88,8 @@ class Source(models.Model):
     def __str__(self):
         if self.author != "" and self.orig_filename != "":
             return "{}, stored by {}".format(self.orig_filename, self.author)
+        elif self.orig_filename != "":
+            return "{}".format(self.orig_filename, self.author)
```

Comment:
I think f-strings are more readable: `f"{self.orig_filename}"` - and it doesn't really make sense to pass the author if its empty, does it?

Reply:
Oups, true about the author, this is what I get for copy/pasting, tsss.